src/edu/stanford/nlp/ie/machinereading/GenericDataSetReader.java

```diff
@@ -444,7 +444,7 @@ private Tree funkyFindLeafWithApproximateSpan(Tree root, String token, int index
     // but it does happen (VERY RARELY) on some weird web text that includes SGML tags with spaces
     // TODO: does this mean that somehow tokenization is different for the parser? check this by throwing an Exception in KBP
     logger.severe("GenericDataSetReader: WARNING: Failed to find head token");
-    logger.severe("  when looking for " + token + " at pos " + index + " plus upto " + approximateness + " in tree: " + root.pennString());
+    logger.finest("  when looking for " + token + " at pos " + index + " plus upto " + approximateness + " in tree: " + root.pennString());
```

Comment:
I guess this transformation should be OK. There is a log invocation with a similar log message above. They should have consistent log levels.